ai_edge_torch/generative/layers/attention.py

```diff
@@ -299,30 +301,42 @@ def forward(
       mask: Optional[torch.Tensor] = None,
       input_pos: Optional[torch.Tensor] = None,
   ):
-    B, T, E = x.size()  # batch size, sequence length, embedding dimensionality (n_embd)
+    """Forward function of the CrossAttention layer.
+
+    Args:
+      x (torch.Tensor): the target tensor, with shape [B, target_seq_len, ...]
+      y (torch.Tensor): the source tensor, with shape [B, source_seq_len, ...]
+      rope (Tuple[torch.Tensor, torch.Tensor]): the input rope tensor.
+      input_pos (torch.Tensor): the optional input position tensor.
```

Comment:
for rope, the comment needs to mention it's optional. and also add `mask` here?

Reply:
done